doc/reference.rst

```diff
@@ -8,3 +8,20 @@ Sphinx-Gallery Reference
 
 .. autodata:: sphinxgallery.__version__
     :annotation: Sphinx Gallery current version
+
+.. include:: modules/generated/sphinxgallery.__version__.examples
+.. raw:: html
+
+        <div style='clear:both'></div>
```

Comment:
Is this clearing needed and if yes should it be added to the .examples file directly?

Also maybe not in this PR but it'd be great to use sphinx templates as scikit-learn is doing in order to introduce the .examples automatically and have an example within sphinx-gallery showing this feature.